package.json

```diff
@@ -240,10 +241,12 @@
       "<rootDir>/__playwright__"
     ],
     "transform": {
-      "^.+\\.svg$": "jest-transform-stub"
+      "^.+\\.svg$": "jest-transform-stub",
+      "^.+\\.(ts|js)$": "ts-jest"
     },
     "transformIgnorePatterns": [
-      "/node_modules/openid-client/"
+      "<rootDir>/node_modules/openid-client/",
+      "<rootDir>/node_modules/string-strip-html/$"
```

Comment:
What was it that required these changes?

Reply:
Jest. If you look in the description, you will see the error you get without these additions.

Reply:
[documentation from jest on esm modules](https://jest-archive-august-2023.netlify.app/docs/ecmascript-modules)